OpenRA.Mods.Common/Projectiles/InstantHit.cs

```diff
@@ -66,16 +66,15 @@ public InstantHit(InstantHitInfo info, ProjectileArgs args)
 
 		public void Tick(World world)
 		{
+			// If GuidedTarget has become invalid due to getting killed the same tick,
+			// we need to set target to args.PassiveTarget to prevent target.CenterPosition below from crashing.
+			if (target.Type == TargetType.Invalid)
+				target = Target.FromPos(args.PassiveTarget);
+
 			// Check for blocking actors
 			WPos blockedPos;
 			if (info.Blockable)
```

Comment:
Moving the invalid check outside this if leaves us with
```
if ()
{
    if ()
       /* do stuff */
}
```
which breaks our style rules for braces and nested ifs. Can you please combine these?

Reply:
Improved it a bit. Hope I understood you correctly.